metricshub-agent/src/main/java/org/sentrysoftware/metricshub/cli/service/PrettyPrinterService.java

```diff
@@ -68,6 +69,11 @@
 @Data
 public class PrettyPrinterService {
 
+	// Constant for red color ANSI escape sequence
+	private static final String ANSI_RED = "\u001B[31m";
```

Comment:
Remove these constants. If you need to use colors, please use the `org.fusesource.jansi` library.